guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -256,6 +287,21 @@ if exists user-class and option user-class = "iPXE" {
 ----
 +
 If you use an isolated network, use a {SmartProxyServer} URL with TCP port `8000`, instead of the URL of {ProjectServer}.
+. In creating a system entry to build it using iPXE workflow, ensure selecting the following as the PXELoader for BIOS and UEFI-based builds in the *Operating System* tab of the Host.
++
+For BIOS Based systems, The PXELoader can be selected as *None* or *iPXE Embedded*.
++
+For UEFI Based systems, The PXELoader must be selected as *None*.
++
+iPXE keeps looping while configuring it to work with the Foreman discovery method for UEFI systems.
+To avoid it, follow the below procedure:
```

Comment:
This looks either like a separate step or like an *Important* admonition to me.

```suggestion
iPXE keeps looping while configuring it to work with the Foreman discovery method for UEFI systems.
To avoid it, enter the following commands:
```